src/app/hooks/useViewTracker/index.tsx

```diff
@@ -37,6 +37,7 @@ const getComponentViewTracker = (eventTrackingData?: EventTrackingData) => {
     groupTracker,
     itemTracker,
     viewThreshold,
+    alwaysInView,
```

Comment:
Would `viewThreshold = 0` achieve similar behaviour to this flag? We could check against it to assume that the component is "always in view" if `viewThreshold = 0`. 

Maybe having this as an explicit option is clearer though?